src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -123,9 +123,23 @@
             tracked by MUX#4382
         -->
         <mux:TeachingTip x:Name="WindowIdToast"
-                         Title="{x:Bind WindowIdForDisplay}"
+                         Title="{x:Bind WindowIdForDisplay, Mode=OneWay}"
                          x:Load="False"
                          IsLightDismissEnabled="True"
-                         Subtitle="{x:Bind WindowNameForDisplay}" />
+                         Subtitle="{x:Bind WindowNameForDisplay, Mode=OneWay}" />
+
+        <mux:TeachingTip x:Name="RenameFailedToast"
+                         x:Uid="RenameFailedToast"
+                         IsLightDismissEnabled="True" />
+
+        <mux:TeachingTip x:Name="WindowRenamer"
+                         x:Uid="WindowRenamer"
+                         Title="{x:Bind WindowIdForDisplay, Mode=OneWay}"
+                         ActionButtonClick="_WindowRenamerActionClick">
+            <mux:TeachingTip.Content>
+                <TextBox x:Name="WindowRenamerTextBox"
+                         Text="{x:Bind WindowName, Mode=OneWay}" />
+            </mux:TeachingTip.Content>
+        </mux:TeachingTip>

```

Comment:
Super surprised this workaround worked. I thought we'd run into the `ContentDialog` input bug.

I'd have to play with the branch a bit, but does it feel right? Like, the `TeachingTip` won't dismiss on you? If you click outside of it, does it get dismissed?

Reply:
I was SHOCKED that input works for TeachingTips and not for ContentDialogs. Maybe b/c TeachingTip is in MUX, not WUX?

oh god does input work in _MUX_ ContentDialogs?